client/pages/_app.js

```diff
@@ -20,6 +20,10 @@ function App({ Component, pageProps }) {
         a:hover {
           color: ${pageProps.isDarkTheme ? theme.dtColors.linkHover : theme.ltColors.linkHover}
         }
+
+        .bold {
+          font-weight: bold;
```

Comment:
@mwhitman189 nit: unless it's a utility class that won't change largely, better to use semantic class names.